src/components/scenes/newAccount/NewAccountTosScene.tsx

```diff
@@ -77,13 +78,60 @@ export const NewAccountTosScene = (props: Props) => {
 
   const handleNextPress = () => {
     logEvent(`Signup_Terms_Agree_and_Create_User`)
-    dispatch(
-      createUser({
-        username,
-        password,
-        pin
-      })
-    )
+    const { username, password, pin } = route.params
+    if (pin == null) throw new Error('No PIN set')
+    const { context } = imports
+    dispatch({
+      type: 'NAVIGATE',
+      data: {
+        name: 'newAccountWait',
+        params: {
+          title: s.strings.great_job,
+          message: s.strings.hang_tight + '\n' + s.strings.secure_account
+        }
+      }
+    })
+    setTimeout(async () => {
```

Comment:
While we haven't enabled the lint rule fully in this repo, you are not allowed to pass an `async` function to `setTimeout`. 

Option 1: Move this code into a `handleCreateAccount` helper, then do `setTimeout(() => handleCreateAccount(...).catch(error => { /* your existing catch block */ })  )`

Option 2: Make `handleNextPress` an `async` function (this is allowed because `<MainButton>` is async-aware). Then replace the `setTimeout` with an `await snooze(300)` line.

Reply:
Let's go with option 1, since that is more similar to the light account refactor that comes later.